timemachine/fe/rbfe.py

```diff
@@ -605,17 +609,31 @@ def estimate_relative_free_energy(
     assert len(keep_idxs) <= len(lambda_schedule)
     combined_prefix = get_mol_name(mol_a) + "_" + get_mol_name(mol_b) + "_" + prefix
     try:
-        return estimate_free_energy_given_initial_states(
+        res = estimate_free_energy_given_initial_states(
             initial_states, protocol, temperature, combined_prefix, keep_idxs
         )
+        if image_traj:
+            for i, idx in enumerate(keep_idxs):
+                res.frames[i] = image_frames(res.initial_states[idx], res.frames[i], res.boxes[i])
```

Comment:
Can you update absolute hydration too: https://github.com/proteneer/timemachine/blob/master/timemachine/fe/absolute_hydration.py#L245 ?

Reply:
or move this into estimate_free_energy_given_initial_states

Reply:
Moved in https://github.com/proteneer/timemachine/pull/905/commits/afaa10d72ca59511699e7d6bddcf96bc91ff242d